autoload/lsp/omni.vim

```diff
@@ -136,7 +136,7 @@ function! s:get_completion_result(data) abort
         let l:incomplete = l:result['isIncomplete']
     endif
 
-    let l:matches = map(l:items, {_, item -> {'word':item['label'],'menu':lsp#omni#get_kind_text(item),'dup':1,'icase':1}})
+    let l:matches = map(l:items, {_, item -> {'word':item['insertText'],'abbr':item['label'],'menu':lsp#omni#get_kind_text(item),'dup':1,'icase':1}})
```

Comment:
`insertText` is optional. Some servers may not provide it. Would to use has_key to check if it exists.

Reply:
would be good if this logic could be moved here.

https://github.com/prabirshrestha/asyncomplete-lsp.vim/blob/master/plugin/asyncomplete.vim#L94-L104